### PR TITLE
Portfolilo Toolbar and headline section

### PR DESCRIPTION
**Pull request by MoRaad97**
This pull contains :
- Render all elements normally using the Normalize.css file and 
- Master CSS file
- Add fonts 
- Add toolbar and headline section HTML code and set the container for the mobile version.
- Declaring global CSS variables and adding CSS for toolbar and headline section
- Fix the image overflow issue.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <img src="imgs/github.png" alt="github" />
+            <img src="imgs/linkedin.png" alt="linkedin" />
+            <img src="imgs/hand.png" alt="none" />
+            <img src="imgs/twitter.png" alt="twitter" />
+            <img src="imgs/M.png" alt="M" />
+          </div>
+        </div>
+      </div>
+      <div class="image">
+        <img src="imgs/Illustration.png" alt="background" />
```
Comment (danielufeli) [inline index.html:56]: - [x] Nice work done on the headline section, but kindly move the background image to the extreme right and make it not to cover the text so it matches the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <span></span>
+            <span></span>
+          </span>
+        </div>
+      </div>
+    </nav>
+    <!-- Toolbar end -->
+    <!--  headline section start -->
+    <section id="headline">
+      <div class="headline-container container">
```
Comment (danielufeli) [inline index.html:37]: - [x] Kindly reduce the space between the toolbar and the title so it matches the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio Website: Mohammed Raad</title>
+  </head>
+  <body>
+    <!-- Toolbar start -->
+    <nav id="nav">
```
Comment (danielufeli) [inline index.html:22]: - [x] Nice work done on the header, but kindly move the nav and logo into a header tag for better use of semantic HTML

File: index.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <div class="text">
+          <h1>Hey There. I’m Mohammed</h1>
+          <h3>I’am a Software Developer</h3>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="social-icon">
+            <img src="imgs/github.png" alt="github" />
```
Comment (danielufeli) [inline index.html:47]: - [x] Your social media icons look good and well aligned but you need to put them in a `ul` tag to group them, and `li` per icon; each icon should be `a` link.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```
Comment (danielufeli) [inline index.html:1]: - [ ] Your Readme Looks Professional, but kindly update the Getting Started Section with the details on how to set up your project locally, you can remove the things that are not required for this project like Run tests, etc.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi Mohammed,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️

## Required Changes ♻️
_Check the comments under the review._

 Alt text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me with @danielufeli. in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — APPROVED** [review]
Hi @MoRaad97 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] On the latest version of your project, your header is not using any styles. This is optional since this problem has come up after the first review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**